Dockerfile

```diff
@@ -1,6 +1,13 @@
+FROM alpine:latest AS builder
+
+RUN apk add --no-cache make go git 
+COPY . ./app
+WORKDIR /app
+RUN make ses-smtpd-proxy
+
 FROM alpine:latest
 
 RUN apk add --no-cache ca-certificates
-ADD ses-smtpd-proxy /
+COPY --from=builder /app/ses-smtpd-proxy /
 
-CMD [ "/ses-smtpd-proxy" ]
+ENTRYPOINT [ "/ses-smtpd-proxy" ]
```

Comment:
I'd prefer to use `CMD` for this rather than `ENTRYPOINT` to make it easier to debug containers. Are you trying to support some other use-case by using `ENTRYPOINT`?

Reply:
Using `ENTRYPOINT` lets us run the container with parameters easily since `CMD` will append. For example: `docker run ses-smtpd-proxy:latest --disable-prometheus`. For debugging I think you can run something like `docker run -it --entrypoint /bin/bash ses-smtpd-proxy:latest`.

Reply:
That makes sense, it's not the way I typically build my images but I see how that's convenient.